modules/konduitWrapper.md

```diff
@@ -0,0 +1,39 @@
+## Konduit video tags wrapper
+
+Konduit Wrapper is a prebid module to generate Konduit wrapped VAST tag URLs for a provided bid or a winning bid.
+
+
+### Setup
+
+```
+pbjs.que.push(function(){
+  pbjs.addAdUnits(videoAdUnit);
+  pbjs.requestBids({
+    timeout : 700,
+    bidsBackHandler : function(bids) {
+      var videoUrl = pbjs.adServers.konduit.buildVastUrl({
+        bid: winnerBid,
+        params: {
+          konduit_id: 'your_konduit_id'
+        }
+      });
+        
+      invokeVideoPlayer(vastTagUrl);
+    }
+  });
+});
```

Comment:
I think the setup example may need to be updated to better explain what is going on.

Where is this `winnerBid` coming from? Good idea to show how it is created from the `bids` coming back.

`invokeVideoPlayer(vastTagUrl);` is this supposed to be `invokeVideoPlayer(videoUrl);`?

Where do all these pieces come together and what exactly is happening in the example. Sure there will be confusion for someone who does not understand `konduit`

Reply:
What does the publisher do with the returned `videoUrl`

Maybe an example test page I can take a look at will help my testing?

Reply:
Robert, we've updated the module documentation file and also created a related snippet at https://jsfiddle.net/edfcgruk/7/.
Please take a look and let us know should you have any questions.